src/main/java/jalau/at18/architects/CardComparatorForResource.java

```diff
@@ -0,0 +1,9 @@
+package jalau.at18.architects;
+
+public class CardComparatorForResource implements Comparator {
+    public boolean compare(Card c1, Card c2) {
+        return ((c1.getColor() == "Gray" && c2.getColor() == "Yellow")
```

Comment:
You could use a constant for the string "Yellow"
example: 
`private static final int YELLOW = "Yellow";`
`private static final int GRAY = "Gray";`
and use it here like
`return ((c1.getColor() == GRAY && c2.getColor() == YELLOW)`
that way the code looks a lot cleaner